health/services.py

```diff
@@ -175,13 +176,28 @@ def _check_disk_space(app):
     return _check_disk_space
 
 
+def _check_zombie_process_count(app):
+    """
+    Returns True if too many zombie processes are running.
+    """
+    processes = get_all_zombies()
+    count = len(processes)
+    msg = "Found %s zombie processes." % count
+
+    max_zombies = app.config.get("MAX_DEFUNCT_PROCESS_COUNT", 0)
```

Comment:
This is absolutely for another day, but this stands out to me now:
since there is supposed to be a default in the app config for every value, I think we should be using `[]` to index into the `app.config` so that we get a crash in the scenario when the programmer has forgotten to add their sane default in the config. Having a default value in case of a lack of default value seems a little crazy.